test/rules/no-consecutive-blank-lines/test.ts.lint

```diff
@@ -14,20 +14,20 @@ class Clazz { // comment
 
 ~nil
         console.log("test");
-~nil                         [consecutive blank lines are disallowed]
+~nil [consecutive blank lines are disallowed]
 
     }
 
 
 ~nil
 }
-~nil  [consecutive blank lines are disallowed]
+~nil [consecutive blank lines are disallowed]
 
 //Begin whitespace
 // The next two lines of "code" contain only tabs or spaces, they are also considered "blank" lines
-~nil                         [consecutive blank lines are disallowed]
         
         
+~ [consecutive blank lines are disallowed]
```

Comment:
What I think is happening here is that the error position is ending up as the first character of the following line or something strange like that. The test system as is struggles a little bit with representing errors nicely visually for whitespace cases like this, but that's a test system quirk not an issue with your PR!